ui/plugin-system/src/components/PluginRegistry/test-plugins/bert/index.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { PanelPlugin } from '../../../../model';
+
+// Dummy plugins to test loading
```

Comment:
Love the new tests!

![bert-ernie](https://user-images.githubusercontent.com/9369625/191310888-0c76a9b9-8b44-47b8-94b3-e4f2eb2ce6d6.gif)